py3status/modules/sysdata.py

```diff
@@ -10,19 +10,38 @@
         (default 75)
     med_threshold: percent to consider CPU or RAM usage as 'medium load'
         (default 40)
+    padding: length of space padding to use on the left
+        (default 0)
+    precision: precision of values
+        (default 2)
+    zone: thermal zone to use. If None try to guess CPU temperature
+        (default None)
```

Comment:
Is there much call for this?  If so maybe we can be more clever so we can have output from multiple 'zones'.